### ISSUE TITLE
mlt templates list from other directory

### ISSUE THREAD

**Issue by s1113950** [IntelAI/mlt#130]
@mas-dse-greina runs into this:
```
ganthony-mac01:Documents ganthony$ mlt templates list
Traceback (most recent call last):
  File "/Users/ganthony/Documents/mlt/.venv/bin/mlt", line 11, in <module>
    sys.exit(main())
  File "/Users/ganthony/Documents/mlt/.venv/lib/python2.7/site-packages/mlt/main.py", line 72, in main
    run_command(args)
  File "/Users/ganthony/Documents/mlt/.venv/lib/python2.7/site-packages/mlt/main.py", line 63, in run_command
    CommandClass(args).action()
  File "/Users/ganthony/Documents/mlt/.venv/lib/python2.7/site-packages/mlt/commands/templates.py", line 35, in action
    templates = self._parse_templates(templates_directory)
  File "/usr/local/Cellar/python/2.7.14_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/Users/ganthony/Documents/mlt/.venv/lib/python2.7/site-packages/mlt/utils/git_helpers.py", line 36, in clone_repo
    shutil.rmtree(destination)
  File "/usr/local/Cellar/python/2.7.14_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py", line 253, in rmtree
    onerror(os.listdir, path, sys.exc_info())
  File "/usr/local/Cellar/python/2.7.14_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py", line 251, in rmtree
    names = os.listdir(path)
OSError: [Errno 2] No such file or directory: '/var/folders/n2/jk3d5w8n14v41py769kbky000000gp/T/tmpptUgGh'
```
it could be due to logic in `__init__.py` here:
```
import os

BASE_DIR = os.path.dirname(os.path.dirname(__file__))

# Relative path to templates within a template repository.
TEMPLATES_DIR = 'mlt-templates'
```
We need to be in either `mlt` or another project dir it looks like that contains `mlt-templates`. We should handle the case when this fails and throw error explaining this to the user.

**Comment by ashahba** [comment]
This was fixed by #147

**Comment by mas-dse-greina** [comment]
Thanks. It works for me now.



On Fri, Apr 13, 2018 at 4:51 PM, Abolfazl Shahbazi <notifications@github.com
